Formula/conduktor-cli.rb

```diff
@@ -1,5 +1,5 @@
 class ConduktorCli < Formula
-  version "0.3.0"
+  version "v0.3.0"
```

Comment:
No v in homebrew version, no matter the underlying go requirements, it should not affect brew version